backend/tests/external_apis/test_c3.py

```diff
@@ -67,13 +111,34 @@ def test_get_submissions_statuses(
 
 def test_get_reports(requests_mock: Mocker, prepare_c3api: tuple[C3Api, str]):
     c3, bearer_token = prepare_c3api
-    report = {"id": 237670, "failed_test_count": 0}
+    c3_api_response = {
+        "id": 237670,
+        "failed_test_count": 0,
+        "test_count": 0,
+        "testresult_set": [],
+    }
+    requests_mock.get(
+        f"https://certification.canonical.com/api/v2/reports/summary/?id__in={c3_api_response['id']}&limit=1",
+        request_headers={"Authorization": f"Bearer {bearer_token}"},
+        json={"results": [c3_api_response]},
+    )
+
     requests_mock.get(
-        f"https://certification.canonical.com/api/v2/reports/summary/?id__in={report['id']}&limit=1",
+        f"https://certification.canonical.com/api/v2/reports/summary/{c3_api_response['id']}/",
```

Comment:
does this endpoint get called? I thought just the previous mocked endpoint will be called

Reply:
Apologies for leaving this here, I used this endpoint before updating C3 to return the test results in the first endpoint as well.